content/en/docs/installation/kubernetes.md

```diff
@@ -39,11 +39,11 @@ are included in a single YAML manifest file:
 Install the `CustomResourceDefinitions` and cert-manager itself:
 
 ```bash
-# Kubernetes 1.15+
-$ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.16.1/cert-manager.yaml
+# Kubernetes 1.16+
```

Comment:
Shall we also update the notes (below) explaining why k8s < 1.16 is now required for multi-version CRDs?